llama_index/agent/context_retriever_agent.py

```diff
@@ -147,8 +147,11 @@ def _get_tools(self, message: str) -> List[BaseTool]:
         return self._tools
 
     def chat(
```

Comment:
what's the motivation for using mode to toggle streaming vs. not here?

Also I don't see how mode is passed to the super class?

Reply:
Another oversight. The notebook tests don't use streaming mode for the context retriever agent, so didn't catch this. Well spotted - fixed now.